lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,24 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
-      origin_branch = current_branch
-      run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
-      run_command_with_label "git checkout #{origin_branch}"
+    def update_destination(options = {})
+      feature_branch = options[:feature_branch]
+      base_branch = options[:base_branch]
+      remote = options[:remote] || 'origin'
+      run_command_with_label "git checkout #{base_branch}"
+      run_command_with_label "git pull #{remote} #{base_branch}"
+      run_command_with_label "git checkout #{feature_branch}"
+    end
+
+    def update_feature_branch(options = {})
+      base_branch = options[:base_branch]
+      update_destination(options)
+
+      # update feature branch in case there are multiple authors and remote changes
+      update_current_branch
```

Comment:
I think we want to push up after the merge, so let's just call the pull directly so we can avoid that here `run_command_with_label "git pull origin #{current_branch}"`